chemprop/args.py

```diff
@@ -219,6 +226,9 @@ class TrainArgs(CommonArgs):
     """Dimensionality of hidden layers in MPN."""
     depth: int = 3
     """Number of message passing steps."""
+    mpn_shared: bool = False
+    """Whether to use the same message passing neural network for all input molecules
+    Only relevant if :code:'number_of_molecules > 1'"""
```

Comment:
Need to change single quotes to backticks to get the ReadTheDocs formatting to work correctly.
```suggestion
    Only relevant if :code:`number_of_molecules > 1`"""
```